packages/loader/test-loader-utils/README.md

```diff
@@ -24,12 +24,12 @@ package for more information including tools to convert between version schemes.
 To get started, install the package by running the following command:
 
 ```bash
-npm i @fluid-internal/test-loader-utils
+npm i @fluid-private/test-loader-utils
```

Comment:
Not for this PR, but we should figure out a way to remove this kind of instructions from packages that aren't published.

Reply:
That is configurable in the pragma that is used to generate them. Good catch! Agreed that it isn't urgent for these PRs, but would be a good cleanup. Same with the API docs notices.

Reply:
Opened https://github.com/microsoft/FluidFramework/pull/18191 to do that.